src/js/components/SkipLinks/SkipLinks.js

```diff
@@ -35,29 +37,38 @@ const SkipLinks = ({ children, id, messages }) => {
   return (
     <Layer
       id={id}
-      position={showLayer ? 'top' : 'hidden'}
+      position={showLayer ? theme.skipLinks.position : 'hidden'}
       ref={layerRef}
       onFocus={onFocus}
       onBlur={onBlur}
+      tabIndex={0}
+      modal={false}
+      bodyTarget="first" // prepend the Layer on the body container
+      // modal={false} will take the full screen a small breakpoint
+      responsive={false}
     >
-      <Box pad={{ horizontal: 'medium' }}>
-        <Heading level={2}>{messages.skipTo}:</Heading>
-        <Box direction="row" align="center" pad={{ bottom: 'medium' }}>
-          {children.map((element, index) =>
-            cloneElement(element, {
-              key: `skip-link-${index}`,
-              onClick: removeLayer,
-            }),
+      {showLayer && (
+        <Box {...theme.skipLinks.container}>
+          {messages.skipTo && (
+            <Text {...theme.skipLinks.text}>{messages.skipTo}</Text>
           )}
+          <Box align="center">
```

Comment:
What if the theme wants a different alignment?

I'm getting nervous that we're putting more and more theming layout and styling in the JSON theme when they would be better handled via JSX markup. Just thinking out loud.

Reply:
We can wait until a need arises.